sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -65,9 +68,9 @@ class EventHubConsumerClient(ClientBase):
      The failed internal partition consumer will be closed (`on_partition_close` will be called if provided) and
      new internal partition consumer will be created (`on_partition_initialize` will be called if provided) to resume
      receiving.
-    :keyword float idle_timeout: Timeout in seconds, after which the underlying connection will close
-     if there is no further activity. By default the value is None, meaning that the service determines when to
-     close an idle connection.
+    :keyword float idle_timeout: Timeout, in seconds, after which this client will close the underlying connection
+     if there is no further activity. By default the value is None, meaning that this client doesn't close but the service
+     determines when to close an idle connection.
```

Comment:
How about this:?

    :keyword float idle_timeout: Timeout in seconds, after which the client will close the underlying connection
     if there is no further activity. By default the value is None, meaning that the client won't track the connection activity and the service determines when to close an idle connection.

Reply:
Changed to Anna's comment